app/components/buttons/ToggleSwitch/ToggleSwitch.module.css

```diff
@@ -1,10 +1,7 @@
 .toggleSwitch {
   display: inline-block;
   vertical-align: middle;
-  margin-right: 0.5em;
-
-  margin-bottom: 10px;
-  width: 85px;
+  margin: 0 0 0.5em 10px;
```

Comment:
Any reasons for using em and px?

Reply:
Actually, it was a mistake. My original intention was to add more margin to the left and remove it from the right while keeping the bottom. I've fixed it now, thanks for the test.